CMakeLists.txt

```diff
@@ -184,6 +184,9 @@ include(cmake/Fluid.cmake)
 message("\nConfiguring src")
 add_subdirectory(src)
 
+# Testing / Reader
+add_subdirectory(Tests)
+
```

Comment:
With the re-org to move most of this code into the `src` directory, is this needed anymore?

Reply:
I don't think so